src/GoogleMaps.js

```diff
@@ -55,10 +55,15 @@ class GoogleMaps extends EventComponent {
     }
     // googleMapsApi can be async loaded
     /*eslint-disable no-unused-vars */
-    const {containerProps, googleMapsApi, ...googleMapsConfig} = props;
+    const {containerProps, googleMapsApi, bounds, ...googleMapsConfig} = props;
     /*eslint-enable no-unused-vars */
     var {instance} = this.state;
 
+    if (bounds) {
```

Comment:
As far as I understand it, `fitBounds` actually sets zoom/center. So this is a precaution to prevent the map from jumping around.